RecoLocalTracker/SiPixelClusterizer/python/SiPixelClusterizer_cfi.py

```diff
@@ -23,6 +23,16 @@
     maxNumberOfClusters = cms.int32(-1), # -1 means no limit.
 )
 
+# phase1 pixel
+from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
+phase1Pixel.toModify(siPixelClusters,
+  VCaltoElectronGain      = cms.int32(47),             # L2-4: 47 +- 4.7
+  VCaltoElectronGain_L1   = cms.untracked.int32(50),   # L1:   49.6 +- 2.6
+  VCaltoElectronOffset    = cms.int32(-60),            # L2-4: -60 +- 130
+  VCaltoElectronOffset_L1 = cms.untracked.int32(-670), # L1:   -670 +- 220
```

Comment:
this is not allowed to be untracked as it modify physics objects
btw I suggest to move to fillDescriptions